.github/workflows/meta-balena-esr.yml

```
@@ -107,7 +107,7 @@ jobs:
             os_version=$(git tag --sort -version:refname | grep "v${va[0]}\." | head -n1)
           fi
           os_version="${os_version:1}"
-          ov_arr=( "${os_version//./ }" )
+          ov_arr=( ${os_version//./ } )
```

```suggestion
          IFS="." read -r -a ov_arr <<< "${current_os_version}"
```
https://www.shellcheck.net/wiki/SC2206